frontend/src/scenes/authentication/signup/signupForm/panels/SignupPanel2.tsx

```diff
@@ -43,6 +44,7 @@ export function SignupPanel2(): JSX.Element | null {
                     center
                     htmlType="submit"
                     data-attr="signup-submit"
+                    onClick={() => setSignupPanel2ManualErrors({})}
```

Comment:
should we clear errors when they touch the field, or on submit?

Reply:
I wanted also to support the case where the server is down, so this supports that case as well. We also don't know what input the error pertains to if it does so we couldn't link the error to a specific input.